### PR TITLE
Updated info in strings.xml

### PR DESCRIPTION
**Pull request by KartikShankhavaram**

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -3,6 +3,8 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/LNMIIT_Android_App.iml" filepath="$PROJECT_DIR$/LNMIIT_Android_App.iml" />
+      <module fileurl="file://$PROJECT_DIR$/LNMIIT_Android_App.iml" filepath="$PROJECT_DIR$/LNMIIT_Android_App.iml" />
```
Comment (vishwesh3) [inline .idea/modules.xml]: Move this file to `.gitignore`

Comment (vishwesh3) [reply .idea/modules.xml]: in separate commit

File: app/build.gradle [hunk 2]
```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.3"
+    buildToolsVersion '25.0.0'
```
Comment (vishwesh3) [inline app/build.gradle]: Why this?

File: build.gradle [hunk 3]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.2'
+        classpath 'com.android.tools.build:gradle:2.3.3'
```
Comment (vishwesh3) [inline build.gradle]: Why this?

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -1,6 +1,6 @@
-#Mon Dec 28 10:00:20 PST 2015
+#Sun Oct 08 10:32:46 IST 2017
```
Comment (vishwesh3) [inline gradle/wrapper/gradle-wrapper.properties]: Why this?
If this is automatic, please add this file to `.gitignore` too.

**Comment by vishwesh3** [comment]
And please add more info in commit message.
like for `Updated info` add which info you have updated.

**Comment by vishwesh3** [comment]
And please include `Fix:#issue_number` this PR solves so that issue is automatically closed after PR merges.

**Comment by KartikShankhavaram** [comment]
OK

File: README.md [hunk 5]
```diff
@@ -2,15 +2,14 @@
...
 
 This version will contains:
 - Redesigning of entire application using Material Desing.
 - Adding navigaton Drawer for easier access of various features of application.
 - High quality code with better code coverage and scalability.
 
-##Mentors: 
-- **[Vishwajeet Srivastava] (https://github.com/vjs3)**
+## Mentors: 
+- **[Vishwajeet Srivastava](https://github.com/vjs3)**
```
Comment (saketkumar95) [inline README.md]: Vishwajeet Srivastava is no longer mentoring this project. Kindly remove this. :stuck_out_tongue_winking_eye:

Comment (KartikShankhavaram) [reply README.md]: So who should be added as mentors?

Comment (saketkumar95) [reply README.md]: Leave with Aditya Jain only

**Pull request closed without merging**